### PR TITLE
use less size and more readable

### PR DESCRIPTION
**Pull request by Playit3110**
Einfach etwas kleiner und simpler

### CODE REVIEW

File: src/scripts/placeDE-overlay.user.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // ==UserScript==
 // @name         r/placeDE Template
 // @namespace    http://tampermonkey.net/
-// @version      6
+// @version      5
```
Comment (arnehude) [inline src/scripts/placeDE-overlay.user.js]: Version missmatch

File: src/scripts/placeDE-overlay.user.js [hunk 2]
```diff
@@ -10,19 +10,17 @@
 // @downloadURL  https://github.com/PlaceDE-Official/place-overlay/raw/main/src/scripts/placeDE-overlay.user.js
 // ==/UserScript==
 
-var overlayImage = null;
-if (window.top !== window.self) {
-    window.addEventListener('load', () => {
-        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("div.layout").querySelector("garlic-bread-canvas").shadowRoot.querySelector("div.container");
+let overlayImage = null;
```
Comment (arnehude) [inline src/scripts/placeDE-overlay.user.js]: using let on root level is not needed.

File: src/scripts/placeDE-overlay.user.js [hunk 3]
```diff
@@ -10,19 +10,17 @@
...
+if(window.top !== window.self) {
+    window.addEventListener("load", function() {
+        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("garlic-bread-canvas").shadowRoot.querySelector(".container");
         overlayImage = document.createElement("img");
+        function updateImage() {
```
Comment (arnehude) [inline src/scripts/placeDE-overlay.user.js:21]: if removing the antipattern, then remove the function as a whole as long as it is not needed twice (code shortening)

File: src/scripts/placeDE-overlay.user.js [hunk 4]
```diff
@@ -10,19 +10,17 @@
...
+            overlayImage.src = "https://place.army/overlay_target.png?" + Date.now()
+        }
         updateImage();
-        overlayImage.style = `position: absolute;left: 0;top: 0;image-rendering: pixelated;width: 1000px;height: 1000px;`;
+        setInterval(updateImage, 30000);
```
Comment (arnehude) [inline src/scripts/placeDE-overlay.user.js]: i would add a variable for this for better overview

File: src/scripts/placeDE-overlay.user.js [hunk 5]
```diff
@@ -10,19 +10,17 @@
...
 // ==/UserScript==
 
-var overlayImage = null;
-if (window.top !== window.self) {
-    window.addEventListener('load', () => {
-        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("div.layout").querySelector("garlic-bread-canvas").shadowRoot.querySelector("div.container");
+let overlayImage = null;
+if(window.top !== window.self) {
+    window.addEventListener("load", function() {
+        const canvasContainer = document.querySelector("garlic-bread-embed").shadowRoot.querySelector("garlic-bread-canvas").shadowRoot.querySelector(".container");
```
Comment (arnehude) [inline src/scripts/placeDE-overlay.user.js:19]: Great shortening!

File: src/scripts/placeDE-overlay.user.js (hunk 3 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js:21]: it is needed for the interval

File: src/scripts/placeDE-overlay.user.js (hunk 1 above)
Comment (arnehude) [reply src/scripts/placeDE-overlay.user.js]: resolved during review

File: src/scripts/placeDE-overlay.user.js (hunk 5 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js:19]: sorry, is just some kink in my programming

File: src/scripts/placeDE-overlay.user.js (hunk 3 above)
Comment (arnehude) [reply src/scripts/placeDE-overlay.user.js:21]: oh, i missed that.

File: src/scripts/placeDE-overlay.user.js (hunk 2 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js]: true

File: src/scripts/placeDE-overlay.user.js (hunk 4 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js]: for the style?

Comment (arnehude) [reply src/scripts/placeDE-overlay.user.js]: For the User to better notice.

If you use a variable such as 'refreshTimer' a user knows what this number is about even if they don't use a ide with intellisense or completion.

Best practice  would also be describing the function but in this case its obsolete.

Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js]: true, but then i would also make style its own var

File: src/scripts/placeDE-overlay.user.js (hunk 2 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js]: now its var

File: src/scripts/placeDE-overlay.user.js (hunk 4 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js]: all made to variable

File: src/scripts/placeDE-overlay.user.js (hunk 5 above)
Comment (Playit3110) [reply src/scripts/placeDE-overlay.user.js:19]: changed back to smaller size

**Review by arnehude — APPROVED** [review]

**Comment by TheDevMinerTV** [comment]
I've taken these and a couple more refactorings and put them into #26 

Thank you for your work 🫡

**Pull request closed without merging**